src/reducers/index.js

```diff
@@ -972,13 +972,16 @@ export const onStoreRouterHistory = (state, action) => {
   };
 };
 
-const onSetServerInfo = (state, action) => ({
-  ...state,
-  serverInfo: {
-    network: action.payload.network,
-    version: action.payload.version,
-  },
-});
+const onSetServerInfo = (state, action) => {
+  const network = action.payload.network.startsWith('testnet') ? 'testnet' : action.payload.network;
```

Comment:
In which cases does the network come with a suffix? And why does it happen only on testnet?

Maybe a comment explaining this is a good idea to help future readers.

Reply:
Oh, ok! I forgot for a moment that we have this suffix on the testnets.

Reply:
I removed this change and made this treatment where we create the network name, since it is how we used to do it.